furnace/related_worker.py

```
@@ -0,0 +1,151 @@
+        # print(result._get_default_filename)
+        # if int(result.published.year)>=2022:
+        # if check_relevant(result.title,result.summary,['CLIP','knowledge distillation']) == 'Y':
+        print(i, result.title, result.entry_id, result.published, )
+        print(result.summary)
+        print('\n\n')
+        rst.append(result)
+    return
+
+def simple_query_s2():
+    rst = []
+    query = f'all:"infrared" AND all:"small" AND all:"Target detection"'
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)